$NetBSD: patch-ab,v 1.1 2000/05/25 10:16:46 wiz Exp $

Override gnome-specific file installation, and make it find pkgsrc's libintl.

--- configure.orig	Fri Apr 14 21:51:24 2000
+++ configure	Thu May 25 11:49:23 2000
@@ -1982,6 +1982,7 @@
   rm -f conf.gdkimlibtest
 
 
+if false; then
   # Extract the first word of "gnome-config", so it can be a program name with args.
 set dummy gnome-config; ac_word=$2
 echo $ac_n "checking for $ac_word""... $ac_c" 1>&6
@@ -2018,6 +2019,7 @@
   echo "$ac_t""no" 1>&6
 fi
 
+fi
   
 
 if test $GNOME_CONFIG != no; then
@@ -3006,7 +3008,7 @@
 #include "confdefs.h"
 #include <libintl.h>
 EOF
-ac_try="$ac_cpp conftest.$ac_ext >/dev/null 2>conftest.out"
+ac_try="$ac_cpp $CFLAGS conftest.$ac_ext >/dev/null 2>conftest.out"
 { (eval echo configure:3011: \"$ac_try\") 1>&5; (eval $ac_try) 2>&5; }
 ac_err=`grep -v '^ *+' conftest.out | grep -v "^conftest.${ac_ext}\$"`
 if test -z "$ac_err"; then
